apps/explorer/src/components/NetworkSelector/NetworkSelector.styled.ts

```diff
@@ -25,12 +23,12 @@ export const OptionsContainer = styled.div<{ width: number }>`
   left: 15px;
   top: 54px;
   background: ${({ theme }): string => theme.bg1};
-  border: ${(): string => `1px solid ${fadedGreyishWhiteOpacity}`};
+  border: 1px solid rgba(141, 141, 169, 0.3);
```

Comment:
Can we have the color from the theme variables instead of hard coding it here?

cc @fairlighteth

Reply:
Ahhh this is the explorer!
Right, we don't have the same nice work there as we have on cowswap

Reply:
But we should work towards that, given we have a mono repo now :)